recipes/console_bridge/all/patches/disable-pkgconfig-export.patch

```diff
@@ -0,0 +1,57 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -72,54 +72,6 @@
+ install(FILES ${PROJECT_BINARY_DIR}/${PROJECT_NAME}_export.h
+   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME})
+ 
+-if(WIN32 AND NOT CYGWIN)
```

Comment:
does it break the windows build? Otherwise I would not patch it, but remove pc files as post package step instead, it will keep the recipe even simpler.